dbt_platform_helper/utils/versioning.py

```diff
@@ -104,7 +104,7 @@ def get_github_released_version(repository: str, tags: bool = False) -> Tuple[in
     return parse_version(package_info["tag_name"])
 
 
-def get_platform_helper_versions() -> PlatformHelperVersions:
+def get_platform_helper_versions(local_and_latest_only=False) -> PlatformHelperVersions:
```

Comment:
I wonder whether this could be named to better express the intent? Maybe something like
```suggestion
def get_platform_helper_versions(include_desired_versions=False) -> PlatformHelperVersions:
```

Reply:
I've changed the name about half a dozen times :-D
Local and latest only does express the intent, which is to get just the local and latest versions. include_desired_versions seems weird as when false it reads "I don't want my desired versions"

Reply:
include_required_versions?

Reply:
for_local_version_check?

Reply:
include_project_versions?

Reply:
That last one is good. Go with that.